commit 3cb59581a860f5f542b37aed214f74b3f2ccfa32
Author: Mark Rutland <mark.rutland@arm.com>
Date:   Tue Jul 19 11:23:37 2016 +0100

    ARM: 8588/1: localise objcopy flags
    
    We currently define OBJCOPYFLAGS in the top-level arm Makefile, and thus
    these flags will be passed to all uses of objcopy, kernel-wide, for
    which they are not explicitly overridden. The flags we set are intended
    for converting a few ELF files into raw binaries, and thus the flags
    chosen are problematic for some other uses which do not expect a raw
    binary result, e.g. the upcoming lkdtm rodata test:
    
      http://www.openwall.com/lists/kernel-hardening/2016/06/08/2
    
    This patch localises the objcopy flags such that they are only used for
    the cases we require them for today.
    
    Signed-off-by: Mark Rutland <mark.rutland@arm.com>
    Acked-by: Kees Cook <keescook@chromium.org>
    Tested-by: Laura Abbott <labbott@redhat.com>
    Cc: Russell King <linux@armlinux.org.uk>
    Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>

diff --git a/arch/arm/boot/Makefile b/arch/arm/boot/Makefile
index bdc1d5af03d2..50f8d1be7fcb 100644
--- a/arch/arm/boot/Makefile
+++ b/arch/arm/boot/Makefile
@@ -11,6 +11,8 @@
 # Copyright (C) 1995-2002 Russell King
 #
 
+OBJCOPYFLAGS	:=-O binary -R .comment -S
+
 ifneq ($(MACHINE),)
 include $(MACHINE)/Makefile.boot
 endif